blackbox/docs/general/builtins/scalar.rst

```diff
@@ -706,9 +706,11 @@ If no ``format_string`` is given the default format will be used::
     +-----------------------------+
     SELECT 1 row in set (... sec)
 
+
 Timezone
 ........
 
+.. _scalar-timezone:
```

Comment:
I think this should go to line 738 https://github.com/crate/crate/pull/8959/files#diff-c1428c761c17d1bb4ed67b8e27361a96R738

Reply:
Ok, it makes more sense as it is the explicit place where the function is defined